.github/workflows/build-release.yml

```diff
@@ -21,3 +21,17 @@ jobs:
           asset_path: ${{ steps.build.outputs.zip_path }}
           asset_name: woocommerce.zip
           asset_content_type: application/zip
+  update-code-reference:
+    if: github.event.release.prerelease == false && github.event.release.draft == false && github.repository_owner == 'woocommerce'
+    name: Update Code Reference
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Invoke Code Reference build and deploy workflow
+        uses: aurelien-baudet/workflow-dispatch@v2
+        with:
+          workflow: GitHub Pages deploy
+          repo: woocommerce/code-reference
+          token: ${{ secrets.CUSTOM_GH_TOKEN }}
+          ref: refs/heads/trunk
```

Comment:
Just a note: This ref is from the code reference repository, it's required a existing ref and inputs to make it run properly.